add: instruction for different binaries (#244)
availproject/op-evm: docs/demo.md

```
Copy the first json url from the table. (in our case it's `http://127.0.0.1:49601/`).

## Setting up DevNet on amd64 machine

To set up a DevNet on an amd64 machine, run the following:

> Run on the root directory:

Reviewer: ```suggestion
```

```
Copy the first json url from the table. (in our case it's `http://127.0.0.1:49601/`).

## Setting up DevNet on amd64 machine

> To set up a DevNet on amd64 machine follow these steps:

Reviewer: ```suggestion
To set up a DevNet on an amd64 machine, run the following:
```

> 1. Clone the github repo on your local machine

Reviewer: I think we can just provide the command based on the Makefile
